test/univariate/discrete/categorical.jl

```diff
@@ -57,6 +57,12 @@ for p in Any[
     @test pdf.(d, support(d)) == p
     @test pdf.(d, 1:k) == p
 
+    @test cf(d, 0) ≈ 1.0
+    @test cf(d, 1) ≈ p' * Distributions.cis.(1:length(p))
```

Comment:
IIRC `cis` is in Base and exported:

```suggestion
    @test cf(d, 1) ≈ p' * cis.(1:length(p))
```

Possibly a bit nicer (also below) would be to use `dot`:

```suggestion
    @test cf(d, 1) ≈ dot(p, cis.(1:length(p))
```
However, possibly this requires to add a `using LinearAlgebra`.